aiida/cmdline/commands/cmd_group.py

```diff
@@ -332,6 +332,13 @@ def group_copy(source_group, destination_group):
     """Add all nodes that belong to source group to the destination group (which may or may not exist)."""
     from aiida import orm
 
-    dest_group = orm.Group.objects.get_or_create(label=destination_group, type_string=source_group.type_string)[0]
+    dest_group, created = orm.Group.objects.get_or_create(label=destination_group, type_string=source_group.type_string)
+
+    # Issue warning if destination group is not empty and get user confirmation to continue
+    if not created and not dest_group.is_empty:
+        echo.echo_warning('Destination group<{}> already exists and is not empty.'.format(destination_group))
```

Comment:
Let's use `group.label` in the echo'd messages. Same few lines below

Reply:
I thought the idea was to use the general group representation, since it also displays information about the group, the user, etc. However, I agree it seems cleaner to have simply the label, and if a user wishes more information they can use `verdi group show`.